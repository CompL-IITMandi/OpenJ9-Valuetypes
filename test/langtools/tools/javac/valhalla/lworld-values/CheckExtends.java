/*
 * @test /nodynamiccopyright/
 * @summary Values may not extend
 *
 * @compile/fail/ref=CheckExtends.out -XDrawDiagnostics CheckExtends.java
 */

final inline class CheckExtends extends Object {
}
