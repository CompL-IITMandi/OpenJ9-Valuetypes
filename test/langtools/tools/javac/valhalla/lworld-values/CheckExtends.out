CheckExtends.java:8:14: compiler.err.value.may.not.extend
1 error
